script/check-size

```diff
@@ -128,67 +128,67 @@ size_nrf52840()
         "UDP_FORWARD=1"
     )
 
-    rm -rf "${OT_TMP_DIR}"
+    rm -rf "$OT_TMP_DIR"
 
     # new commit
-    mkdir -p "${OT_TMP_DIR}/b"
-    git archive "${OT_SHA_NEW}" | tar x -C "${OT_TMP_DIR}/b"
+    mkdir -p "$OT_TMP_DIR/b"
+    git archive "$OT_SHA_NEW" | tar x -C "$OT_TMP_DIR/b"
 
-    (cd "${OT_TMP_DIR}/b" \
-        && ./bootstrap \
-        && make -f examples/Makefile-nrf52840 "${flags[@]}")
+    (cd "$OT_TMP_DIR/b" &&
+        ./bootstrap &&
+        make -f examples/Makefile-nrf52840 "${flags[@]}")
 
     # old commit
     if [[ "${GITHUB_ACTIONS+x}" ]]; then
-        git fetch --depth 1 --no-recurse-submodules origin "${OT_SHA_OLD}"
+        git fetch --depth 1 --no-recurse-submodules origin "$OT_SHA_OLD"
     fi
 
-    mkdir -p "${OT_TMP_DIR}/a"
-    git archive "${OT_SHA_OLD}" | tar x -C "${OT_TMP_DIR}/a"
-    (cd "${OT_TMP_DIR}/a" \
-        && ./bootstrap \
-        && make -f examples/Makefile-nrf52840 "${flags[@]}")
+    mkdir -p "$OT_TMP_DIR/a"
+    git archive "$OT_SHA_OLD" | tar x -C "$OT_TMP_DIR/a"
+    (cd "$OT_TMP_DIR/a" &&
```

Comment:
Google's shell style guide suggests the original style guide. https://google.github.io/styleguide/shellguide.html#pipelines
Not sure why.